### PR TITLE
UC01 done - Basic set up, shows a class on localhost

### PR DESCRIPTION
**Pull request by PTMGitHub**

### CODE REVIEW

File: application/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+				.ce4 { font-family:Calibri; background-color:#ed7d31; vertical-align:top; text-align:center ! important; margin-left:0cm; writing-mode:font-size:11pt; font-style:normal; text-shadow:none; text-decoration:none ! important; font-weight:bold; }
+				.ce5 { font-family:Liberation Sans; background-color:transparent; vertical-align:top; margin-left:0cm; writing-mode:}
+				.ce6 { font-family:Calibri; background-color:#808080; vertical-align:top; margin-left:0cm; writing-mode:color:#000000; font-size:11pt; font-style:normal; text-shadow:none; text-decoration:none ! important; font-weight:bold; }
+				.ce7 { font-family:Liberation Sans; background-color:#ed7d31; vertical-align:top; margin-left:0cm; writing-mode:}
+				.ce8 { font-family:Liberation Sans; background-color:transparent; vertical-align:top; margin-left:0cm; writing-mode:}
+				.ce9 { font-family:Calibri; background-color:#ed7d31; vertical-align:top; margin-left:0cm; writing-mode:font-size:11pt; font-style:normal; text-shadow:none; text-decoration:none ! important; font-weight:normal; }
```
Comment (StephenThomson) [inline application/templates/index.html:53]: You could probably pull the css out to a seperate file and then reference

File: application/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+					</td>
+					<td class="ce17">
+						<p>03:10</p>
+					</td>
+					<td  class="ce10">
+						<p>sprints</p>
+					</td>
+					<td class="ce21">
+						<p>Start seated. 0:28 up resis stand up jog. 0:50 leave resis sit andsprint(18 sec).</p>
+						<p>1:27 up resis stand up jog. 1:49 leave resis sit and sprint.</p>
```
Comment (StephenThomson) [inline application/templates/index.html:295]: A good first step towards using a db to store your info would be to store this in a python object and then use jinja magic to bring it in.

**Review by StephenThomson — COMMENTED** [review]
Looks great, the HTML looks large and manually created but I know you have plans to move away from that.

Good job :+1:

**Comment by PTMGitHub** [comment]
Actually, the HTML is how LibreOffice converted the spreadsheet into html.

I didnt wanna waste time at that stage faffing with HTML.

It's part of this ticket lol

Paul Marsh 07834536757 paulmarsh@hotmail.com

________________________________

**Pull request merged**